bedrock/firefox/templates/firefox/browsers/quantum.html

```diff
@@ -0,0 +1,52 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% add_lang_files "firefox/browsers/chromebook" %}
+
+{% from "macros-protocol.html" import hero, picto_card with context %}
+
+{% extends "firefox/base/base-protocol.html" %}
+
+{% block page_title %}{{ ftl('the-latest-firefox') }}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('quantum') }}
+{% endblock %}
+
+{% block content %}
+
+{% call hero(
+  title=ftl('the-latest-firefox'),
+  class='mzp-has-image mzp-t-product-firefox mzp-t-dark',
+  image_url='img/firefox/enterprise/fx-browser-img.svg',
+  include_highres_image=True,
+  include_cta=True,
+  heading_level=1
+) %}
+<p>{{ ftl('firefox-quantum-was') }}</p>
+<p><a href="https://blog.mozilla.org/blog/2017/11/14/introducing-firefox-quantum/">{{ ftl('learn-more-about') }}</a></p>
+{% endcall %}
+
+<div class="mzp-l-content">
+  <section class="mzp-c-emphasis-box">
+    <ul class="mzp-l-card-third">
+      {{ picto_card(
+        title=ftl('privacy-first'),
+        desc=ftl('firefox-doesnt-spy'),
+        class='privacy') }}
+
+      {{ picto_card(
+        title=ftl('super-fast'),
+        desc=ftl('get-speed-and'),
+        class='fast') }}
+
+      {{ picto_card(
+        title=ftl('always-evolving'),
+        desc=ftl('find-out-about',  attrs='href="%s"'|safe|format(url('firefox'))),
```

Comment:
Suggest just passing `url` is as a param here, unless there's a need for multiple attributes?